test/integration/kubernetes_deploy_test.rb

```diff
@@ -9,10 +9,12 @@ def test_full_hello_cloud_set_deploy_succeeds
 
     assert_logs_match_all([
       "Deploying ConfigMap/hello-cloud-configmap-data (timeout: 30s)",
-      %r{Deploying Pod/unmanaged-pod-[-\w]+ \(timeout: 60s\)}, # annotation timeout override
+      /Deploying resources:/,
+      %r{- Pod/unmanaged-pod-1-[-\w]+ \(timeout: 60s\)}, # annotation timeout override
+      %r{- Pod/unmanaged-pod-2-[-\w]+ \(timeout: 60s\)}, # annotation timeout override
       "Hello from the command runner!", # unmanaged pod logs
```

Comment:
This test will need a slight re-work its failing when the following reasonable output happens:

```

[INFO][2019-03-07 21:45:37 +0000]	Logs from Pod/unmanaged-pod-2-k63a7981-70f8292e container 'hello-cloud':
--
  | [INFO][2019-03-07 21:45:37 +0000]		Hello from the second command runner!
  | [INFO][2019-03-07 21:45:37 +0000]
  | [INFO][2019-03-07 21:45:37 +0000]	Logs from Pod/unmanaged-pod-1-k63a7981-70f8292e container 'hello-cloud':
  | [INFO][2019-03-07 21:45:37 +0000]		Hello from the command runner!
  | [INFO][2019-03-07 21:45:37 +0000]
  | [INFO][2019-03-07 21:45:37 +0000]	Successfully deployed in 3.6s: Pod/unmanaged-pod-2-k63a7981-70f8292e, Pod/unmanaged-pod-1-k63a7981-70f8292e

```

Reply:
Nothing in that log output is asserted here. Could you show me the test failure? Even artificially delaying the execution of unmanaged-pod-1, I can't make this test fail.

Reply:
Sorry about that I didn't include enough. The issue really isn't about the execution time its about the order k8s-deploy logs things.
```

Failure:
--
  | KubernetesDeployTest#test_full_hello_cloud_set_deploy_succeeds [/usr/src/app/test/integration/kubernetes_deploy_test.rb:10]
  | Minitest::Assertion: 'Hello from the command runner!' not found in the expected sequence in the following logs:
  | [INFO][2019-03-07 21:45:26 +0000]
  | [INFO][2019-03-07 21:45:26 +0000]	------------------------------------Phase 1: Initializing deploy------------------------------------

...


[INFO][2019-03-07 21:45:34 +0000]
  | [INFO][2019-03-07 21:45:34 +0000]	Deploying resources:
  | [INFO][2019-03-07 21:45:34 +0000]	- Pod/unmanaged-pod-2-k63a7981-70f8292e (timeout: 60s)
  | [INFO][2019-03-07 21:45:34 +0000]	- Pod/unmanaged-pod-1-k63a7981-70f8292e (timeout: 60s)
  | [INFO][2019-03-07 21:45:37 +0000]	Logs from Pod/unmanaged-pod-2-k63a7981-70f8292e container 'hello-cloud':
  | [INFO][2019-03-07 21:45:37 +0000]		Hello from the second command runner!
  | [INFO][2019-03-07 21:45:37 +0000]
  | [INFO][2019-03-07 21:45:37 +0000]	Logs from Pod/unmanaged-pod-1-k63a7981-70f8292e container 'hello-cloud':
  | [INFO][2019-03-07 21:45:37 +0000]		Hello from the command runner!
  | [INFO][2019-03-07 21:45:37 +0000]
  | [INFO][2019-03-07 21:45:37 +0000]	Successfully deployed in 3.6s: Pod/unmanaged-pod-2-k63a7981-70f8292e, Pod/unmanaged-pod-1-k63a7981-70f8292e
  | [INFO][2019-03-07 21:45:37 +0000]
  | [INFO][2019-03-07 21:45:37 +0000]
  | [INFO][2019-03-07 21:45:37 +0000]	----------------------------------Phase 4: Deploying all resources----------------------------------
 
...
```

Reply:
In that case, I'll just wait for https://github.com/Shopify/kubernetes-deploy/pull/441 before rebasing this.

Reply:
I went ahead and rebased this on top of #441